server/app.js

```diff
@@ -45,6 +45,11 @@ app.use(cookieParser());
 
 app.use('/static', express.static(path.join(__dirname, '../client/dist')));
 
+// Special static path for service worker
+// since it can interfere with requests to it's path or below
+app.use('/sw.js', express.static(path.join(__dirname, '../client/dist/sw.js')));
```

Comment:
I don't understand why this is a separate route. Could you flesh out the comment with a bit more detail?